src/main/resources/templates/mdBenefitsFlow/householdIncome.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="~{fragments/head :: head(title=#{household-income.title})}"></head>
+<body>
+<div class="page-wrapper">
+  <div th:replace="~{fragments/toolbar :: toolbar}"></div>
+  <section class="slab">
+    <div class="grid">
+      <div th:replace="~{fragments/goBack :: goBackLink}"></div>
+      <main id="content" role="main" class="form-card spacing-above-35">
+        <th:block th:replace="~{fragments/icons :: personWithDollarSign}"></th:block>
+        <th:block
+            th:replace="~{fragments/cardHeader :: cardHeader(header=#{household-income.header})}"/>
+        <th:block
+            th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
+          <th:block th:ref="formContent">
+            <div class="form-card__content">
+              <p th:text="#{household-income.p1}"></p>
+              <p th:text="#{household-income.p2}"></p>
+              <ul class="list--bulleted">
+                <li th:text="#{household-income.li1}"></li>
+                <li th:text="#{household-income.li2}"></li>
+                <li th:text="#{household-income.li3}"></li>
+                <li th:text="#{household-income.li4}"></li>
+                <li th:text="#{household-income.li5}"></li>
```

Comment:
Just noting that the li6 is not showing up here.  The line about the barber, construction, etc... 

<img width="282" alt="Screenshot 2024-03-01 at 5 47 00 PM" src="https://github.com/codeforamerica/mdpilot/assets/7966375/bd982748-2697-43f4-afa7-b0f132a514c0">